.github/workflows/e2e.yaml

```diff
@@ -92,3 +115,11 @@ jobs:
           kubectl logs -n $NAMESPACE $POD -c manager
           echo "previous pod logs (if any)"
           kubectl logs -p -n $NAMESPACE $POD -c manager || true
+
+  end-to-end:
+    runs-on: ubuntu-latest
+    needs:
+      - e2e-tests
+    steps:
+      - run: |
+          echo "Placeholder job for required checks"
```

Comment:
If we're going with the matrix approach, I can disable the end-to-end test as a required check - just let me know

Reply:
Got it, makes sense!

Reply:
Maybe we can replace the output to 'All E2E tests ran successfully'

Reply:
Done :)